frontend/app/routes/public/routes.ts

```diff
@@ -554,8 +554,28 @@ export const routes = [
           },
           {
             id: 'public/renew/$id/adult-child/exit-application',
-            file: 'routes/public/renew/$id/adult-child/exit-application.tsx',
-            paths: { en: '/:lang/renew/:id/adult-child/exit-application', fr: '/:lang/renew/:id/adulte-enfant/quitter-demande' },
+            file: 'routes/public/renew/$id/child/exit-application.tsx',
+            paths: { en: '/:lang/renew/:id/child/exit-application', fr: '/:lang/renew/:id/adulte-enfant/quitter-demande' },
```

Comment:
This should be unchanged right? There's a route (`id` = `public/renew/$id/child/exit-application`) defined below with the same `file` and `paths` value.